ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/InternalPaywallView.kt

```diff
@@ -63,6 +64,7 @@ private fun getPaywallViewModel(offering: Offering?, listener: PaywallViewListen
             LocalContext.current.applicationContext.toAndroidContext(),
             offering,
             listener,
+            MaterialTheme.colors,
```

Comment:
We should check if light/dark theme colors get automatically updated in the default paywall. My strong bet is that no. In that case, we should maybe store the last state in the view model so we can update the state when it changes, but we can do that in another PR.